File: crypto/src/main/java/net/corda/v5/crypto/merkle/MerkleProofType.java
package net.corda.v5.crypto.merkle;

import net.corda.v5.base.annotations.CordaSerializable;

/**
 * {@link MerkleProofType} represents what type of {@link MerkleProof} was created.
 */
@CordaSerializable
public enum MerkleProofType {
    AUDIT, SIZE
}

File: crypto/src/main/kotlin/net/corda/v5/crypto/merkle/MerkleProofType.kt
package net.corda.v5.crypto.merkle

import net.corda.v5.base.annotations.CordaSerializable

/**
 * [MerkleProofType] represents what type of [MerkleProof] was created.
 */
@CordaSerializable
enum class MerkleProofType {

    AUDIT, SIZE
}